#  Robot message 

float64[] target_pose
float64[] joints_position_reference
